src/test/java/org/cloudfoundry/autosleep/dao/repositories/RouteBindingRepositoryTest.java

```diff
@@ -0,0 +1,147 @@
+package org.cloudfoundry.autosleep.dao.repositories;
```

Comment:
Globally each functionnalities of the test should be splitted into single test (Given/When/Then) that test each assumptions of the dao